apps/web/vibes/soul/sections/error/index.tsx

```diff
@@ -1,14 +1,14 @@
-import { ButtonLink } from '@/vibes/soul/primitives/button-link';
+import { Button } from '@/vibes/soul/primitives/button';
 
-interface Link {
+interface Action {
   label: string;
-  href: string;
+  onRetry: () => void;
```

Comment:
nit: I'd rename this `onClick` since `onRetry` is too specific. There's no retry logic in this component